### PR TITLE
docs(misc): change http:// to https://

### PR DESCRIPTION
**Pull request by beeman**
## Current Behavior
I found a few typos (and `http://` that could be `https://`) while looking through the Nx source code. 

## Expected Behavior
It has fewer typos.

### CODE REVIEW

File: packages/create-nx-workspace/bin/create-nx-workspace.ts [hunk 1]
```diff
@@ -277,9 +277,6 @@ async function normalizeArgsMiddleware(
   try {
     let thirdPartyPreset: string | null;
 
-    // Node options
-    let docker: boolean;
```
Comment (beeman) [inline packages/create-nx-workspace/bin/create-nx-workspace.ts:281]: Unused variable

**Review by isaacplmann — APPROVED** [review]

**Review by mandarini — APPROVED** [review]

**Review by AgentEnder — APPROVED** [review]

**Comment by marleypowell** [comment]
This appears to have broken some stuff when running storybook commands

```
Error: no schema with key or ref "https://json-schema.org/schema"
    at Ajv.validate (C:\repo\node_modules\ajv\lib\core.ts:359:21)
    at Ajv.validateSchema (C:\repo\node_modules\ajv\lib\core.ts:515:24)
    at Ajv._addSchema (C:\repo\node_modules\ajv\lib\core.ts:721:30)
    at Ajv.compile (C:\repo\node_modules\ajv\lib\core.ts:384:22)
    at CoreSchemaRegistry._compile (C:\repo\node_modules\@angular-devkit\core\src\json\schema\registry.js:246:35)
    at CoreSchemaRegistry.compile (C:\repo\node_modules\@angular-devkit\core\src\json\schema\registry.js:231:37)
    at name (C:\repo\node_modules\@angular-devkit\architect\src\architect.js:224:43)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by mandarini** [comment]
@marleypowell o no!!! what version of Nx are you using? do you a repro repo??

**Comment by marleypowell** [comment]
It started after this package update.
image

I can try and get a repro repo.

**Comment by mandarini** [comment]
@marleypowell what command are you running that's failing?

a repro repo would be SUPER helpful, thanks!!!!

**Comment by marleypowell** [comment]
> @marleypowell what command are you running that's failing?
> 
> a repro repo would be SUPER helpful, thanks!!!!

`yarn nx storybook <project>`

**Comment by marleypowell** [comment]
I haven't got a repro just yet but I've narrowed it down to the `ng-packagr-lite` executor. 

https://github.com/beeman/nx/blob/65c448727d2b6b00fea27f7f374989712cff8cec/packages/angular/src/executors/ng-packagr-lite/schema.json

**Comment by mandarini** [comment]
Hmm just tried it, I could not reproduce: https://github.com/mandarini/storybook-nx-17.3.1

Do you want to try to delete `node_modules` and install again? Maybe some weird caching issue.

**Comment by marleypowell** [comment]
Yeah I can't reproduce it in a clean repo either. Trying to work out what's different...

It's happening to everyone for the same repository and multiple people have reinstalled `node_modules` but it still happens 😕

**Comment by mandarini** [comment]
I assume you cannot share your repo, right?

**Comment by marleypowell** [comment]
Figured out the problem... `browserTarget` pointing to `build` vs `build-storybook`.

image

**Comment by mandarini** [comment]
Ohh thanks for pointing this out!! I suggest, specifically for Angular + Storybook, you check out the official https://storybook.js.org/tutorials/intro-to-storybook/angular/en/get-started/ documentation. The Nx Storybook docs are fine (I've written them :P) but sometimes I may have missed some update in the official Storybook docs! I should remove some of the Nx+Storybook docs that are overlapping, I guess 😬

**Comment by marleypowell** [comment]
Some of the [storybook docs](https://storybook.js.org/docs/configure/styling-and-css) say to use `build` 😁

image

**Comment by mandarini** [comment]
Ohhhh I see!!!! So you got it working now with `build` or `build-storybook`? Glad you got it working in any case hehehe

**Comment by mandarini** [comment]
Perfect, thanks!!

**Comment by marleypowell** [comment]
Oh I forgot to mention, it has to be a buildable lib.

https://github.com/storybookjs/storybook/issues/16977#issuecomment-1010957495

**Comment by mandarini** [comment]
oh wow interesting! nice to see my comments from 2 years ago 😅 thanks for linking these

**Comment by marleypowell** [comment]
@mandarini should I create a new issue for this?

**Comment by marleypowell** [comment]
> oh yes sure please do and I'll assign myself!

#21636

**Pull request merged**